src/content/lesson/what-is-an-array-define-array.es.md

```diff
@@ -211,48 +209,48 @@ console.log(y); // [14, 3, 'a', 3245, 234, 52, 345, 3, 45, 23, 77]
 
-## Clasificando los Arreglos
+## Ordenando arrays
 ***
 
-Es muy común la necesidad de ordenar arreglos. Por ejemplo: ordenar una lista de estudiantes por nombre. Tienes dos funciones para ordenar en JavaScript:
+Es muy común la necesidad de ordenar arrays. Por ejemplo: ordenar una lista de estudiantes por nombre. Tienes dos funciones para ordenar en JavaScript:
 
 Sort y Reverse
 
-Ellos hacen lo mismo, excepto que Reverse lo hace al revés. Ambos clasifican usando la lógica de comparación de cadenas, lo que significa que la cadena "25" es más grande que "100", porque "2" es más grande que "1".
+Ellos hacen lo mismo, excepto que Reverse lo hace al revés. Ambos ordenan usando la lógica de comparación de string, lo que significa que el string "25" es más grande que "100", porque "2" es más grande que "1".
 
 ```javascript
-var fruits = ["Banana", "Orange", "Apple", "Mango"];
-    fruits.sort();
-    console.log(fruits); //[ 'Apple', 'Banana', 'Mango', 'Orange' ] 
-    fruits.reverse();
-    console.log(fruits); //[ 'Orange', 'Mango', 'Banana', 'Apple' ]
+var frutas = ["Banana", "Naranja", "Manzana", "Mango"];
+    frutas.sort();
+    console.log(frutas); //[ 'Manzana', 'Banana', 'Mango', 'Naranja' ] 
+    frutas.reverse();
+    console.log(frutas); //[ 'Naranja', 'Mango', 'Banana', 'Manzana' ]
 ```
 
-### Clasificando Números
+### Ordenando números
 
-Si deseas ordenar números reales, o si deseas usar cualquier otro tipo de inicio de sesión para ordenar arreglos, debes usar una "función de comparación".
+Si quieres ordenar números reales, o si deseas usar cualquier otro tipo de inicio de sesión para ordenar arrays, debes usar una "función de comparación".
 
 Tienes que definir una función que se encargue de las comparaciones. La función sort llamará a tu función en cada comparación y permitirá que tu función decida quién es el primero entre los dos elementos que se comparan.
 
 ```javascript
-var points = [40, 100, 1, 5, 25, 10];
-    points.sort(function(a, b){return a – b});
-    console.log(points); //[ 1, 5, 10, 25, 40, 100 ]
+var puntos = [40, 100, 1, 5, 25, 10];
+    puntos.sort(function(a, b){return a – b});
+    console.log(puntos); //[ 1, 5, 10, 25, 40, 100 ]
 ```
 
 ### Clasificando Objetos
 
-Ahora que conocemos la función de comparación, podemos usarla para indicar a la función sort cómo ordenar nuestros propios objetos especiales, como aquí, por ejemplo:
+Ahora que conocemos la función de comparación, podemos usarla para indicar a la función sort cómo ordenar nuestros propios objetos especiales, como aquí por ejemplo:
 
 ```javascript
-var cars = [
-    {type:"Volvo", year:2016},
-    {type:"Saab", year:2001},
-    {type:"BMW", year:2010}];
-    cars.sort(function(a, b){return a.year – b.year});
-    console.log(cars); //[ { type: 'Saab', year: 2001 },{ type: 'BMW', year: 2010 },{ type: 'Volvo', year: 2016 } ]
+var autos = [
```

Comment:
The example bellow in the `iframe` takes you to a code that is in english, we should keep this code in english